bin/main.py

```diff
@@ -41,6 +38,12 @@ def main():
     pass
 
 
+def text_extract_utf8(f):
+    try:
+        return unicode(textract.process(f), "utf-8")
+    except UnicodeDecodeError, e:
+        return e
```

Comment:
Could we return the empty string instead? Returning the error message might make it appear that the person's resume has error text.

How about:
```
except UnicodeDecodeError, e:
        return ''
```